com.unity.shadergraph/Documentation~/Blackboard.md

```diff
@@ -1,19 +1,81 @@
 # Blackboard
 
 ## Description
-The Blackboard lists [Properties](https://docs.unity3d.com/Manual/SL-Properties.html) and [Keywords](Keywords) you can use in this graph. It is also where you can edit the path for this Shader Graph Asset or Sub Graph.
-You can move the Blackboard anywhere in the [Shader Graph Window](Shader-Graph-Window). It always maintains the same distance from the nearest corner, even if you resize the window.
+You can use the Blackboard to define, order, and categorize the [Properties](Property-Types) and [Keywords](Keywords) in a graph. From the Blackboard, you can also edit the path for the selected Shader Graph Asset or Sub Graph.
 
+![image](images/blackboardcategories1.png)
 
-## Creating nodes
-Drag a Property or Keyword from the Blackboard to the graph to create a node of that kind. To create a new Property or Keyword, click the **Add (+)** button on the Blackboard's title bar and select a type. For a full list of Property types, see [Property Types](Property-Types).
+## Accessing the Blackboard
+The Blackboard is visible by default, and you cannot hide it. However, you are able to position it anywhere in the [Shader Graph Window](Shader-Graph-Window). It always maintains the same distance from the nearest corner, even if you resize the window.
```

Comment:
Doc says you cannot hide the blackboard, but we have this button that toggles the blackboard's visibility.

![image](https://user-images.githubusercontent.com/13984097/120704241-37b3b100-c46b-11eb-8c7c-1ca3b04b9b6c.png)

Did you mean that the blackboard can't be dragged off the graph and lost?

Reply:
fixed, thanks!